### PR TITLE
revamped ID gui added installfont func and assets

### PR DESCRIPTION
**Pull request by Altoid0**
pls approve need food to feed family

### CODE REVIEW

**Comment by Altoid0** [comment]
Ok fr though, revamped Team ID GUI to include icon and background basically the entire thing is changed. With this came new assets (ico and png files for background and button). Also designed it to use a non-default font that needs to be installed. The installation has its own func in release_windows.go. GUI and font install func both tested locally. Build succeeded for all changes.

**Review by safinsingh — APPROVED** [review]
lgtm. can another windoze person pls test locally?

File: assets/Raleway/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+DISCLAIMER
+THE FONT SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
+OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE
+COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
+DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
+OTHER DEALINGS IN THE FONT SOFTWARE.
```
Comment (aaditgupta21) [inline assets/Raleway/OFL.txt:93]: Can we remove this

File: assets/Raleway/README.txt [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+License
+-------
+Please read the full license text (OFL.txt) to understand the permissions,
+restrictions and requirements for usage, redistribution, and modification.
+
+You can use them freely in your products & projects - print or digital,
+commercial or otherwise. However, you can't sell the fonts on their own.
+
+This isn't legal advice, please consider consulting a lawyer and see the full
+license for all details.
```
Comment (aaditgupta21) [inline assets/Raleway/README.txt]: Can we remove this

**Pull request merged**